website/docs/contributing/ADRs/back-end/specificity-db-columns.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "ADR: Specificity in database column references"
+---
+
+## Background
+
+We recently experienced an issue where a database migration that introduced a new column resulted in ambiguity errors in our application, which highlighted the need for clearer SQL query standards. Currently, our queries often reference columns without specifying their parent tables, leading to potential ambiguity in complex queries that join multiple tables. This issue becomes more pronounced during database schema changes and migrations, where ambiguity in column references can lead to hard-to-anticipate runtime errors.
+
+## Decision
+
+To mitigate these risks, we will adopt a standard of explicitly specifying the full table name or alias for each column in our SQL queries. This standard is not just about improving readability, but is crucial for avoiding ambiguity in queries, especially when performing joins between tables. The decision to use the full table name or an alias will be left to the discretion of the developer, with a focus on maximizing clarity and maintainability.
+
```

Comment:
Quick example of preferred code and discouraged syntax could make it easier to read, process and understand.

Reply:
Addressed in https://github.com/Unleash/unleash/pull/5499/commits/c5f7fd38c4e8352084a6787e0e5ada127a8d4378 - Let me know if it helps 👍